css/long-form.css

```diff
@@ -0,0 +1,783 @@
+/*------------------------------------------------------------------
+[Master Stylesheet]
+Version:	1.0
+Date published:	04/03/2016
+Designed by: Punal C
+Primary use: Long form template
+[Table of contents]
+1. General
+	1.1 Reset Bootstrap
+	1.2 Reset TNA Base
+2. Sections
+3. Container
+4. Components
+5. Navigation
+-------------------------------------------------------------------*/
+
+/*--------------------
+1. General
+--------------------*/
+
+body {
+    margin: 0;
+}
+
+/*--------------------
+1.1 Reset bootstrap
+--------------------*/
+
+ol, ul {
+    list-style: none;
+}
+
```

Comment:
There was a reason at the time to remove list styles as this was going to be read as stories and list styles were not needed. Which is why i removed them. I feel it should still be there, as i feel editors should have a choice to use list if they wish to. So i will add them back in.